closed/openj9_version_info.h.in

```diff
@@ -30,5 +30,6 @@
 #define J9VERSION_STRING          "@VERSION_STRING@"
 #define OPENJDK_SHA               "@OPENJDK_SHA@"
 #define OPENJDK_TAG               "@OPENJDK_TAG@"
+#define J9JVM_VERSION_STRING      "@OPENJ9_VERSION_STRING@"
```

Comment:
This would be easier to follow throughout the code base if the macro were called `OPENJ9_VERSION_STRING`.

Reply:
The reason to choose `J9` instead of `OPENJ9` here is that both `OpenJ9` and internal build reference this definition. `J9` fits better in these two scenarios.